package.json

```diff
@@ -17,7 +17,7 @@
   },
   "scripts": {
     "postinstall": "bower install && bundle install",
-    "build": "grunt build --force && mv public docs",
+    "build": "grunt build --production && rsync -aqr public/ docs/",
```

Comment:
your `mv` command was moving the public folder `inside` of the docs folder, not replacing it (because `docs` already exists. this `rsync -aqr source/ dest/` will replace and overwrite everything that gets rebuilt by grunt from public to docs but not delete custom things (like the `CNAME` file you added).

Reply:
I fixed this locally afterwards but never pushed it - thanks!